packages/dd-trace/test/fixtures/telemetry/config_prefix_block_list.json

```diff
@@ -0,0 +1,243 @@
+  "trace.googlespanner_analytics_sample_rate",
+  "trace.guzzle_enabled",
+  "trace.guzzle_analytics_enabled",
+  "trace.guzzle_analytics_sample_rate",
+  "trace.laminas_enabled",
+  "trace.laminas_analytics_enabled",
+  "trace.laminas_analytics_sample_rate",
+  "trace.laravel_enabled",
+  "trace.laravel_analytics_enabled",
+  "trace.laravel_analytics_sample_rate",
+  "trace.laravelqueue_enabled",
+  "trace.laravelqueue_analytics_enabled",
+  "trace.laravelqueue_analytics_sample_rate",
+  "trace.logs_enabled",
+  "trace.logs_analytics_enabled",
+  "trace.logs_analytics_sample_rate",
+  "trace.lumen_enabled",
+  "trace.lumen_analytics_enabled",
+  "trace.lumen_analytics_sample_rate",
+  "trace.memcache_enabled",
+  "trace.memcache_analytics_enabled",
+  "trace.memcache_analytics_sample_rate",
+  "trace.memcached_enabled",
+  "trace.memcached_analytics_enabled",
+  "trace.memcached_analytics_sample_rate",
+  "trace.mongo_enabled",
+  "trace.mongo_analytics_enabled",
+  "trace.mongo_analytics_sample_rate",
+  "trace.mongodb_enabled",
+  "trace.mongodb_analytics_enabled",
+  "trace.mongodb_analytics_sample_rate",
+  "trace.mysqli_enabled",
+  "trace.mysqli_analytics_enabled",
+  "trace.mysqli_analytics_sample_rate",
+  "trace.nette_enabled",
+  "trace.nette_analytics_enabled",
+  "trace.nette_analytics_sample_rate",
+  "trace.openai_enabled",
+  "trace.openai_analytics_enabled",
+  "trace.openai_analytics_sample_rate",
+  "trace.pcntl_enabled",
+  "trace.pcntl_analytics_enabled",
+  "trace.pcntl_analytics_sample_rate",
+  "trace.pdo_enabled",
+  "trace.pdo_analytics_enabled",
+  "trace.pdo_analytics_sample_rate",
+  "trace.phpredis_enabled",
+  "trace.phpredis_analytics_enabled",
+  "trace.phpredis_analytics_sample_rate",
+  "trace.predis_enabled",
+  "trace.predis_analytics_enabled",
+  "trace.predis_analytics_sample_rate",
+  "trace.psr18_enabled",
+  "trace.psr18_analytics_enabled",
+  "trace.psr18_analytics_sample_rate",
+  "trace.roadrunner_enabled",
+  "trace.roadrunner_analytics_enabled",
+  "trace.roadrunner_analytics_sample_rate",
+  "trace.sqlsrv_enabled",
+  "trace.sqlsrv_analytics_enabled",
+  "trace.sqlsrv_analytics_sample_rate",
+  "trace.slim_enabled",
+  "trace.slim_analytics_enabled",
+  "trace.slim_analytics_sample_rate",
+  "trace.swoole_enabled",
+  "trace.swoole_analytics_enabled",
+  "trace.swoole_analytics_sample_rate",
+  "trace.symfonymessenger_enabled",
+  "trace.symfonymessenger_analytics_enabled",
+  "trace.symfonymessenger_analytics_sample_rate",
+  "trace.symfony_enabled",
+  "trace.symfony_analytics_enabled",
+  "trace.symfony_analytics_sample_rate",
+  "trace.web_enabled",
+  "trace.web_analytics_enabled",
+  "trace.web_analytics_sample_rate",
+  "trace.wordpress_enabled",
+  "trace.wordpress_analytics_enabled",
+  "trace.wordpress_analytics_sample_rate",
+  "trace.yii_enabled",
+  "trace.yii_analytics_enabled",
+  "trace.yii_analytics_sample_rate",
+  "trace.zendframework_enabled",
```

Comment:
It seems weird that frameworks for other platforms are listed in the Node.js codebase. Will all tracers contain an entry for Next.js?

Reply:
No, because Next.js status isn't sent in the configuration telemetry by default. This file is copied directly from [dd-go](https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_prefix_block_list.json)